### PR TITLE
Update IntentWidget.java

### PR DESCRIPTION
**Pull request by wpride**

### CODE REVIEW

**Comment by wpride** [comment]
http://manage.dimagi.com/default.asp?171026

fixes bug causing crash for intentwidget question types

**Comment by ctsims** [comment]
@wpride I don't understand what your fogbugz link is supposed to be pointing to. Nothing in there mentions intent callouts, is that where the bug came from?

**Comment by wpride** [comment]
updated with correct ticket @ctsims

File: app/src/org/odk/collect/android/widgets/IntentWidget.java [hunk 1]
```diff
@@ -84,7 +84,6 @@ public void makeTextView(FormEntryPrompt prompt) {
         }
 
         // finish complex layout
-        addView(launchIntentButton);
```
Comment (ctsims) [inline app/src/org/odk/collect/android/widgets/IntentWidget.java]: If the bug is that this is null at startup, that also applies to the above condition (s != null, results in a direct dereference), that should also be cleaned up before this is accepted.

**Comment by dcluna** [comment]
Apparently this also solves http://manage.dimagi.com/default.asp?171260

**Comment by wpride** [comment]
@ctsims @dcluna this is ready for review

**Comment by dcluna** [comment]
See also https://github.com/dimagi/commcare-odk/pull/338

**Comment by phillipm** [comment]
This is fixes a 2.22 bug, did these commits actually make it into the commcare_2.22 branch?

**Pull request merged**